### PR TITLE
docs: Updated blurb about installing via npm

### PR DESCRIPTION
**Pull request by Doug-AWS**
npm command installs the latest released version

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license.
j

### CODE REVIEW

**Comment by rix0rrr** [comment]
Not sure this is better. It uses more words to describe something that's pretty obvious anyway.

**Comment by Doug-AWS** [comment]
Actually what I wanted to get across is that you can use it to update your current CDK bits. For those not familiar with npm. So there's no need to uninstall the previous version, which is often the case with other systems.

But if we think this is over-kill, just close it.

**Comment by eladb** [comment]
Maybe we can explicitly call out the upgrade use case instead of just allude to it...

File: README.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ on AWS.
 * For a detailed walkthrough, see [Getting Started] in the AWS CDK [User Guide]
 * See [Manual Installation](./MANUAL_INSTALLATION.md) for installing the CDK from a signed .zip file
 
-Install the [AWS CDK Toolkit] from npm (requires [Node.js ≥ 8.11.x](https://nodejs.org/en/download)):
+Install the [AWS CDK Toolkit] or update your installed version of the |cdk| to the latest version from npm (requires [Node.js ≥ 8.11.x](https://nodejs.org/en/download)):
```
Comment (rix0rrr) [inline README.md]: Can we just phrase as:

```
Install or update the [AWS CDK Toolkit] using npm:
```

And be done with it? The shorter the easier to read.

Comment (Doug-AWS) [reply README.md]: Sure

**Comment by Doug-AWS** [comment]
Hmm, it looks like there's ~300 file changes, likely because I need to rebase.

**Pull request closed without merging**